### PR TITLE
Updated capture_ui_element & validate_ui_element for input tag.

### PR DESCRIPTION
**Pull request by chetanmk98**
## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have added tests to cover my changes.
- [ ] My change requires a change to the documentation.
- [ ] I have Extended README or added documentation/comments where applicable
- [x] Added copyright headers for new files from CONTRIBUTING.md
- [x] My code follows the code style of this project.

- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have updated the [ChangeLog](CHANGELOG.md) following [Semantic Versioning](https://semver.org)

### CODE REVIEW

File: engine/selenium_helper.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def click_button(test_executor: TestExecutor, **kwargs):
 
     """
     locator, locator_value = get_locator_info(**kwargs)
-    logger.info("I'll click a button!")
+    logger.info("I'll Click a button!")
```
Comment (hriddhighosh4) [inline engine/selenium_helper.py:200]: Why is this log changed?

File: engine/selenium_helper.py [hunk 2]
```diff
@@ -238,13 +238,18 @@ def validate_ui_element(test_executor: TestExecutor, **kwargs):
             element = WebDriverWait(test_executor.web_driver, 10).until(
                 EC.visibility_of_element_located((locator, locator_value))
             )
+            elementTag = element.tag_name
+            if elementTag == "input":
```
Comment (hriddhighosh4) [inline engine/selenium_helper.py:242]: Add log here

File: engine/selenium_helper.py [hunk 3]
```diff
@@ -238,13 +238,18 @@ def validate_ui_element(test_executor: TestExecutor, **kwargs):
...
+                element_value = element.get_attribute("value")
+            else:
+                element_value = element.text
```
Comment (hriddhighosh4) [inline engine/selenium_helper.py:245]: Add log

File: engine/selenium_helper.py [hunk 4]
```diff
@@ -238,13 +238,18 @@ def validate_ui_element(test_executor: TestExecutor, **kwargs):
...
             take_screenshot(test_executor.test_id, test_executor.test_yml["name"], test_executor.web_driver)
             if mode == 'equals':
-                validation_result = (value == element.text)
+                validation_result = (value == element_value)
             elif mode == 'contains':
-                validation_result = str(element.text).__contains__(value)
+                validation_result = str(element_value).__contains__(value)
             elif mode == 'not_contains':
-                validation_result = not str(element.text).__contains__(value)
+                validation_result = not str(element_value).__contains__(value)
```
Comment (hriddhighosh4) [inline engine/selenium_helper.py:252]: Have you tested this for the input tag
Validate a prefilled value in input boc

File: engine/selenium_helper.py [hunk 5]
```diff
@@ -442,9 +448,17 @@ def capture_ui_element(test_executor: TestExecutor, **kwargs):
...
-    element = test_executor.web_driver.find_element(locator, locator_value)
-    test_executor.variable_dictionary[save_in] = element.text
-    logger.info("saved text from UI element. variable: {}, value: {}".format(save_in, element.text))
+    element = WebDriverWait(test_executor.web_driver, 10).until(
+        EC.element_to_be_clickable((locator, locator_value))
+    )
+    take_screenshot(test_executor.test_id, test_executor.test_yml["name"], test_executor.web_driver)
+    elementTag = element.tag_name
+    if elementTag == "input":
+        element_value = element.get_attribute("value")
```
Comment (hriddhighosh4) [inline engine/selenium_helper.py:457]: Add log here

File: engine/selenium_helper.py (hunk 1 above)
Comment (chetanmk98) [reply engine/selenium_helper.py:200]: Was updated during a test

File: engine/selenium_helper.py (hunk 4 above)
Comment (chetanmk98) [reply engine/selenium_helper.py:252]: Validated

**Review by hriddhighosh4 — APPROVED** [review]

**Pull request merged**